pkgdb/include/flox/pkgdb/read.hh

```diff
@@ -93,13 +93,25 @@ struct SqlVersions
 
 }; /* End struct `SqlVersions' */
 
+/** @brief SQLite3 schema fields for DbScrapeMeta table. */
+struct ScrapeMeta
+{
+  /**
+   * The hash representing the Scraping rules used to generate the database.
+   * This is checked upon opening and if they differ than the rules expected in
+   * the running version of pkgdb, the datbase is invalidated so it will be
+   * recreated on the *next* invocation.
```

Comment:
nit: this comment is outdated now, right?

Reply:
Yep, nice catch!